### DISCUSSION TITLE
How to read file "Birth" date?

### DISCUSSION THREAD

**Discussion by aborruso** [nushell/nushell#7842] [Q&A] [answered]
Hi,
for some filesystems we have this kind of date information for a file:

```
Access: 2023-01-22 18:54:00.129166583 +0100
Modify: 2023-01-22 18:54:00.059166601 +0100
Change: 2023-01-22 18:54:00.059166601 +0100
 Birth: 2023-01-22 17:51:58.940341716 +0100
```

Is there a way to get Birth date via nu commands? If yes, how to?

Thank you

**Comment by fdncred** [comment, marked as answer]
I believe `Birth` is `created` in nushell that you can see with `ls -l`. nushell has `created`, `accessed`, and `modified`.

**Reply by aborruso** [reply]
Thank you, it works
